app/dockerdwrapper.c

```diff
@@ -79,11 +79,29 @@ static const char *tls_certs[] = {"ca.pem",
                                   "server-cert.pem",
                                   "server-key.pem"};
 
+#define g_main_loop_run_log()                                                  \
```

Comment:
Since they use the global variable 'loop', they are probably not GLib "g_" generic. We could simply call them:

    main_loop_run()
    main_loop_quit()
    
without g_ and _log(), so it's easier to find stray 'g_main_loop' when bringing in new code.